experimental/Schemes/SpaceGerms.jl

```diff
@@ -0,0 +1,276 @@
+#####                 which case this should be referred to the global case, or nonsense
+#####               - for complements of ideals: either I is the unique maximal ideal, 
+#####                 i.e.the complement of the inverted set, or nonsense
+
+function germ_at_point(X::AbsSpec, I::Ideal)
+  Y = SpaceGerm(X, I)
+  restr_map = SpecMor(Y, X, hom(OO(X), OO(Y), gens(OO(Y)), check=false), check=false)
+  return Y, restr_map
+end
+
+function germ_at_point(X::AbsSpec, a::Vector)
+  Y = SpaceGerm(X, a)
+  restr_map = SpecMor(Y, X, hom(OO(X), OO(Y), gens(OO(Y)), check=false), check=false)
+  return Y, restr_map
+end
+
+function germ_at_point(A::MPolyRing, I::Ideal)
+  X = Spec(A)
+  Y = SpaceGerm(X, I)
+  restr_map = SpecMor(Y, X, hom(OO(X), OO(Y), gens(OO(Y)), check=false), check=false)
+  return Y, restr_map
+end
+
+function germ_at_point(A::MPolyRing, a::Vector)
+  X = Spec(A)
+  Y = SpaceGerm(X, a)
+  restr_map = SpecMor(Y, X, hom(OO(X), OO(Y), gens(OO(Y)), check=false), check=false)
+  return Y, restr_map
+end
+
+function germ_at_point(A::MPolyQuo, I::Ideal)
+  X = Spec(A)
+  Y = SpaceGerm(X, I)
+  restr_map = SpecMor(Y, X, hom(OO(X), OO(Y), gens(OO(Y)), check=false), check=false)
+  return Y, restr_map
+end
+
+function germ_at_point(A::MPolyQuo, a::Vector)
+  X = Spec(A)
+  Y = SpaceGerm(X, a)
+  restr_map = SpecMor(Y, X, hom(OO(X), OO(Y), gens(OO(Y)), check=false), check=false)
+  return Y, restr_map
+end
+
+#########################################################################################
+## for convenience of users thinking in terms of local rings
+#########################################################################################
+
+LocalRing = Union{MPolyQuoLocalizedRing{<:Any, <:Any, <:Any, <:Any, 
+                                        <:MPolyComplementOfKPointIdeal},
+                  MPolyLocalizedRing{<:Any, <:Any, <:Any, <:Any, 
+                                     <:MPolyComplementOfKPointIdeal},
+                  MPolyQuoLocalizedRing{<:Any, <:Any, <:Any, <:Any, 
+                                        <:MPolyComplementOfPrimeIdeal},
+                  MPolyLocalizedRing{<:Any, <:Any, <:Any, <:Any, 
+                                     <:MPolyComplementOfPrimeIdeal}
+                 }
+
+
+function SpaceGerm(A::LocalRing)
+  return SpaceGerm(Spec(A))
+end
+
+## and with identity map to keep usage consistent
+function germ_at_point(A::LocalRing)
+  X = SpaceGerm(A)
+  restr_map = SpecMor(X, X, hom(OO(X), OO(X), gens(OO(X)), check=false), check=false)
+  return X, restr_map
+end
+
+### basic functionality for space germs
+
+##############################################################################
+# note: ==, intersect are inherited from Spec
+#       intersect with explicit fallback to Spec and change of return type
+##############################################################################
+
+function issubset(X::AbsSpaceGerm, Y::AbsSpaceGerm)
```

Comment:
```suggestion
function issubset(X::AbsSpaceGerm{<:Any, <:MPolyQuoLocalizedRing}, Y::AbsSpaceGerm{<:Any, <:MPolyQuoLocalizedRing})
```
This should do the job, if I remember correctly...